include/inviwo/core/util/indexmapper.h

```diff
@@ -54,7 +54,7 @@ struct IndexMapper<2, IndexType> {
     }
 
 private:
-    const IndexType dimx;
+    IndexType dimx;
```

Comment:
Why remove const here?

Reply:
To allow the assignment operator to work

Reply:
But, there are no assignment operator :P